src/screens/GroupInfo/elements/ModalAction.js

```diff
@@ -77,7 +88,12 @@ const ModalAction = (props) => {
               </TouchableOpacity>
             )}
             <TouchableOpacity
-              onPress={() => onPress('message-anonymously', selectedUser)}
+              onPress={() => {
+                if (!selectedUser?.allow_anon_dm) {
+                  return SimpleToast.show('This user does not allow messages in Incognito Mode.');
+                }
+                onPress('message-anonymously', selectedUser);
+              }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the changes made here are logical and improve the user experience by providing feedback when an action cannot be performed. The check for `selectedUser?.allow_anon_dm` before triggering the onPress event is a good practice.
```diff
+               onPress={() => {
+                 if (!selectedUser?.allow_anon_dm) {
+                   return SimpleToast.show('This user does not allow messages in Incognito Mode.');
+                 }
+                 onPress('message-anonymously', selectedUser);
+               }}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->